doc/tutorials/make_your_first_quantum_circuit.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>The quotes render strangely in the documentation - perhaps better to make it part of the rest of the paragraph, or perhaps even use a 'note' admonition like above.</p><p><br></p><p>Suggested change: In PennyLane, the term 'device' refers to a the computational object that can apply quantum operations and return an expectation value.</p><p><br></p><p>(removing the backticks around 'device')</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>